### ISSUE TITLE
freehand digitizing not working (symbol greyed out)

### ISSUE THREAD

**Issue by rlzt057** [opengisch/QField#1636]
Hello,

I am trying to work with the new freehand digitizing function in QField beta 1.8.1 using a Samsung Galaxy Tab Active Pro.
The device comes with a stylus pen. The pen can be used to i.e. place vertex points of a line (one by one) - this works. But drawing a continuous line with the stylus via freehand digitizing does not work.
Am I using the function inappropriately or could this be a compatibility issue? 

### Reproduction Steps:
1. Open a QField-project and switch to editing mode.
2. Select the layer to draw lines in.
3. When the stylus pen gets near the display, it is recognized by the device (cursor reacts properly) and in the top-left screen of QField there appears a freehand digitizing symbol with a grey-colored background. 
4. When contacting the stylus to the display and holding it there the freehand digitizing symbol remains grey-colored; moving the stylus across the display has no effect.

### Expected behavior:
The expected behaviour would be that by moving the stylus across the display QField activates freehand editing (green symbol) and listens to the stylus' movement.

### Observed behavior:
See "Reproduction Steps".

### Screenshots and GIFs
Imho providing a screenshot is not necessary for the given problem.

**QField version:** 1.8.1

Thanks in advance!

**Comment by nirvn** [comment]
@rlzt057 , have you tried activating the freehand mode by clicking on the freehand button that appears when your stylus hovers the screen first? 

I.e.:

https://user-images.githubusercontent.com/1728657/106344112-75e16e00-62db-11eb-8a60-f0daccb8c69c.mp4

**Comment by rlzt057** [comment]
Hi nirvn,

thanks for your help and sharing the media.
The shown way of activating didn't work for me using Version 1.8.1. When pointing the stylus on the grey-colored symbol QField placed a vertex point beneath the symbol. The symbol itself didn't respond. So there was no way activating the feature for me (with the device I used).
But I saw that there is a new Version 1.8.2. I tested it and it works properly now (symbol responds to clicking it now before and after digitizing process - in between as well).
Thank you - going to test that feature in detail! 'really looking forward for the next release!
